portfolio/src/main/webapp/style.css

```diff
@@ -148,6 +148,121 @@ hr {
     border: 1.5px solid #c0c0c0;
 }
 
+/* Work Experience */
+
+.work-experience h4 {
+    margin: 2px;
+}
+
+.work-experience .date {
+    color: #676767;
+    margin-top: 5px;
+}
+
+.work-experience .description {
+    text-align: justify;
+    margin: auto;
+}
+
+/* Sunflower Parallax */
+.parallax {
+    background-image: url("../images/sunflowers.jpg");
+    min-height: 200px;
+    /* parallax scrolling effect */
+    background-attachment: fixed;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+    opacity: 0.65;
+}
+
+.parallax p {
+    font-weight: lighter;
+    color: #f4f4f4;
+    position: absolute;
+    left: 0;
+    top: 50%;
+    width: 100%;
+    text-align: center;
+    z-index: 1;
+}
+
+/* "View all Projects" link */
+#projects .projects-link {
+    text-align: right;
+}
+
+#projects a {
+    color: grey;
+    text-decoration-style: solid;
+}
+
+#projects a:hover {
+    color: darkgray;
+}
+
+/* PROJECT CARDS */
+
```

Comment:
nit, Unnecessary newline after comment. Because of new line styles below seem unrelated to comment :D

Reply:
You're right I didn't realize that, I'll get rid of this extra newline right away!